crowbar_framework/app/helpers/form_helper.rb

```diff
@@ -41,6 +41,19 @@ def roles_for_select(selected)
     )
   end
 
+  # Possible values for nodes default file system
+  def fs_for_select(selected)
+    options_for_select(
+      [
+        ["ext3", "ext3"],
```

Comment:
Hm, something like "Think carefully before you change it" looks too discouraging, I'd say...

I think we should leave it this way (without any additional comment) and rather add some more text to Deployment guide... although I'm not sure what else could we write there besides the fact that we recommend btrfs for nova-docker nodes...